boards/espressif/esp32c3_devkitm/Kconfig

```diff
@@ -3,4 +3,5 @@
 
 config HEAP_MEM_POOL_ADD_SIZE_BOARD
 	int
+	default 32768 if BT
```

Comment:
BT and WiFi should be using the runtime heap, so increasing the system heap doesn't make sense now.